app/controllers/tutoring_session_controller.rb

```diff
@@ -19,7 +19,7 @@ def index
 
       start_week += 1.day # Offset by a day due to how time parses out when converted from cookie
     else
-      start_week = Date.today.beginning_of_week(start_day = :sunday).to_datetime
+      start_week = Date.today.beginning_of_week.to_datetime
```

Comment:
I assume it starts on monday by default?

Reply:
Yes